### PR TITLE
Cards (also made it into Nef version)

### PR DESCRIPTION
**Pull request by DeathlyPlays**
"PSGO plugin"

### CODE REVIEW

**Comment by Lord-Haji** [comment]
@DeathlyPlays Just copying code doesnt work out
u did not test out

File: spacialgaze-plugins/cards.js [hunk 1]
```diff
@@ -0,0 +1,908 @@
...
+ * Cards
+ * Avalon's Card System: Credit to Naten, nineage, fender, and everyone who added cards/
+ * All cards should be retrieved here http://www.pokemon.com/us/pokemon-tcg/pokemon-cards/
+ * Cards are organized alphabetically and use a point system
+ * publicids are dex numbers and any unique identifiers (if they're not a Pokemon, do a shorthand version of the card name)
+ * Dex Number, (for multiple pokemon: DEX[lowercase letter, a, b, c, d])
+ ********************/
+const uuid = require('uuid');
+const cards = require('../config/card-data.js');
+let color = require('../config/color');
```
Comment (Lord-Haji) [inline spacialgaze-plugins/cards.js:12]: ..

File: spacialgaze-plugins/cards.js [hunk 2]
```diff
@@ -0,0 +1,908 @@
...
+				if (obj.hasOwnProperty('rarity') && obj.rarity.indexOf(cardRarity[i]) > -1) rareCache[i].push(key);
+			}
+		}
+	}
+	for (let i = 0; i < cardRarity.length; i++) {
+		cleanCard.push(toId(cardRarity[i]));
+	}
+}
+
+global.tourCard = function (tourSize, userid) {
```
Comment (Lord-Haji) [inline spacialgaze-plugins/cards.js:71]: SG ?

**Comment by Lord-Haji** [comment]
No card adding flexibility

**Review by Lord-Haji — COMMENTED** [review]
No Card adding flexibility

**Comment by HoeenCoder** [comment]
This is probably going to get closed due to RAM / Disk Space.

File: spacialgaze-plugins/cards.js (hunk 1 above)
Comment (HoeenCoder) [reply spacialgaze-plugins/cards.js:12]: He dosen't know that we dont have `config/color.js`, @DeathlyPlays youll need to use `SG.hashColor`

**Comment by HoeenCoder** [comment]
@DeathlyPlays please test your code

File: spacialgaze-plugins/cards.js (hunk 1 above)
Comment (Lord-Haji) [reply spacialgaze-plugins/cards.js:12]: Isn't he supposed to rest it before PR

Comment (HoeenCoder) [reply spacialgaze-plugins/cards.js:12]: You don't always test yours either, anyways that my problem not yours.

Comment (Lord-Haji) [reply spacialgaze-plugins/cards.js:12]: I do test **always** But sometimes I may get screwed and forget to test all the stuff

**Pull request closed without merging**